docs/module/routing.md

```diff
@@ -433,11 +433,12 @@ A static route entry's **nexthop** attribute specifies the next hop used to reac
 
 * **ipv4** -- an IPv4 next hop (an address, not a prefix)
 * **ipv6** -- an IPv6 next hop (an address, not a prefix)
+* **gateway** (boolean) -- the next hop is the default gateway. This entry will be resolved into a list containing all directly connected default gateways in the specified next-hop VRF (a node might use more than one default gateway to reach a destination).
 * **node** -- The next hop is the specified node. The node name will be resolved into a list of directly connected next-hops or the control-plane endpoint of a distant node (allowing you to specify recursive static routes).
```

Comment:
What happens if the node has multiple addresses, for example through ```gateway```? Which address gets selected - the interface address I would presume?

Reply:
Ah, the precedence. **node** is before **gateway** which is before **ipv4** / **ipv6**. Also, have to add the **_valid_with** check on **gateway** (and wondering why it doesn't work on **node**)